.travis.yml

```diff
@@ -40,12 +40,18 @@ install:
   - sudo apt-get install -y graphviz
   - pip install git+https://github.com/cylc/cylc-flow/
   - pip install -e .'[all]'
+  - pip install pillow
```

Comment:
Should it be a group in `setup.py`? Maybe `[tutoria]` included by default in `[all]` too? The `get-rainfall.py` script relies on it, from what I could tell. There are other scripts that need `matplotlib` too, but we are ignoring if the package is not found, so I think maybe we don't need matplotlib, just pillow in `setup.py`?

Reply:
I guess it should probably go in `EXTRAS_REQUIRE["test"]`?